homeassistant/components/volvooncall/__init__.py

```diff
@@ -98,13 +98,15 @@
                 vol.Required(CONF_PASSWORD): cv.string,
                 vol.Optional(
                     CONF_SCAN_INTERVAL, default=DEFAULT_UPDATE_INTERVAL
-                ): vol.All(cv.time_period, vol.Clamp(min=MIN_UPDATE_INTERVAL)),
+                ): vol.All(
+                    cv.time_period, vol.Clamp(min=DEFAULT_UPDATE_INTERVAL)
```

Comment:
We should mark these as deprecated with `cv.deprecated`.

Example:
https://github.com/home-assistant/core/blob/b7b965c9c9575e3717eb4bef39809c55c700b7d4/homeassistant/components/elkm1/__init__.py#L146-L155